Index: gst-plugins-bad-0.10.22/autogen.sh
===================================================================
--- gst-plugins-bad-0.10.22.orig/autogen.sh	2011-09-14 18:33:04.372952489 +0900
+++ gst-plugins-bad-0.10.22/autogen.sh	2011-09-14 18:33:15.392951636 +0900
@@ -29,7 +29,7 @@
     ln -s ../../common/hooks/pre-commit.hook .git/hooks/pre-commit
 fi
 
-CONFIGURE_DEF_OPT='--enable-maintainer-mode --enable-gtk-doc'
+CONFIGURE_DEF_OPT='--enable-maintainer-mode'
 
 autogen_options $@
 
